addons/account/models/account_account.py

```diff
@@ -719,6 +726,13 @@ def action_open_related_taxes(self):
             'domain': [('id', 'in', related_taxes_ids)],
         }
 
+    @api.model
+    def get_import_templates(self):
+        return [{
+            'label': _('Import Template for Chart of Accounts'),
+            'template': 'https://docs.google.com/spreadsheets/d/1rneV51mYaA5uTJlQlfYGg9Yy0fYiORu13xiNbK7ANYk/edit?usp=sharing'
```

Comment:
This seems odd, why are we requiring access to google spreadsheets? And AFAIK we can't import from spreadsheets so this seems like it'll just add confusion.

Reply:
I think it's because we can maintain the spreadsheet up-to-date easier (probably). The spreadsheet can be downloaded from there, should be intuitive enough?

Reply:
That seems odd, we can have template files in the repository and manage the history there. I don't see why we need to delegate parts to external systems.

Reply:
@chklop may be able to explain why

Reply:
@mart-e we discussed that with FP and AL. Their argument was at the time that it would avoid storing files and versions in the repo to limit the size, and that adapting / updating the template wouldn't require a new version or a migration.

On a functional level, I can work with both, so it can be a tech-driven decision at this point. If you would like to suggest an alternative, maybe ask AL directly.